File: Task1.Notes.Python/core/repository/db_connector.py
def create(path):
    try:
        file = open(path, 'r')
    except IOError:
        file = open(path, 'w')
        print('Создана новая записная книжка \> ' + path)
    finally:
        file.close()

def read_all(path):
    file = open(path, 'r')
    notes = list()
    for line in file:
        notes.append(line)
    file.close()
    return notes

def save_all(path, notes):
    file = open(path, 'w')
    file.writelines("%s\n" % note for note in notes)
    file.close()

def save_note(path, new_note):
    file = open(path, 'a')
    file.write(new_note + "\n")
    file.close()    

File: Task2.ToysShop.Java/ToysShop/src/main/java/toysshop/repositorium/DBConnector.java
package toysshop.repositorium;

import java.io.*;
import java.util.ArrayList;
import java.util.List;

public class DBConnector {
    private final String path;

    public DBConnector(String path) {
        this.path = path;
        connectDB();
    }
    public void connectDB() {
        try {
            File db = new File(path);
            if (db.createNewFile()) {
                System.out.println("New DB created");
            }
            else {
                System.out.println("DB already exists");
            }
        }
        catch (Exception e) {
            //noinspection ThrowablePrintedToSystemOut
            System.err.println(e);
        }
    }

    public void add(List<String> lines) {
        try (FileWriter writer = new FileWriter(path, true)) {
            for (String line : lines) {
                writer.write(line + '\n');
            }
            writer.flush();
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
    }
    public void saveAll(List<String> lines) {
        try (FileWriter writer = new FileWriter(path, false)) {
            for (String line : lines) {
                writer.write(line + '\n');
            }
            writer.flush();
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
    }
    public List<String> readAll() {
        List<String> lines = new ArrayList<>();
        try {
            FileReader fr = new FileReader(path);
            BufferedReader reader = new BufferedReader(fr);
            String line = reader.readLine();
            if (line != null) {
                lines.add(line);
            }
            while (line != null) {
                line = reader.readLine();
                if (line != null) {
                    lines.add(line);
                }
            }
            fr.close();
        } catch (IOException e) {
            //noinspection CallToPrintStackTrace
            e.printStackTrace();
        }
        return lines;
    }
    public int newIndex(){
        return readAll().size()+1;
    }

    public boolean isExist(String name){
        List<String> storage = readAll();
        for (String line: storage) {
            if (line.contains(name)) return true;
        }
        return false;
    }
}
